src/less/lightbox-dialog/lightbox-dialog.less

```diff
@@ -25,6 +25,27 @@
     .dialog-footer-content();
 }
 
+.lightbox-dialog__image {
+    background-position: center;
+    background-repeat: no-repeat;
+    background-size: cover;
+    border-radius: var(--border-radius-100) var(--border-radius-100) 0 0;
```

Comment:
It looks like for the expressive variation, we're supposed to have `24px` border radius and padding. 
<img width="819" alt="image" src="https://github.com/eBay/skin/assets/1675667/6cd4fb66-0a67-446e-9dc8-0ff933254d64">

Reply:
Good catch on the padding, thanks. That said, it looks to me like the border radius is actually still the same as nonexpressive (16px). Maybe I'll reach out to design about this.

Reply:
Yea, you're right - a bit of an optical illusion. Ok, this is cool. That new`dev` mode is great!